backend/tools/event_sender/src/main.rs

```diff
@@ -82,7 +83,7 @@ async fn app() -> anyhow::Result<()> {
             .add_link(Link::new("Link0", true))
             .add_link(Link::new("Link1", true))
             .add_event(
-                Event::new("Event", "1.0.0")
+                Event::new(type_array[rand::thread_rng().gen_range(0..4)], "1.0.0")
```

Comment:
this conflicts with the event sender pr

Reply:
That should not be there, good catch! I will revert this file to the latest commit on your master.

Reply:
It has been reverted to this commit [f8fd31a](https://github.com/ItJustWorksTM/EiffelVis/commit/f8fd31a77af249366ea7dd00540f928fb2690dd7)